src/sensors/utils.rs

```diff
@@ -240,15 +253,6 @@ impl ProcessTracker {
                                 }
                             };
                         description.insert(String::from("container_id"), container_id.clone());
-                        //let container_id = cg
-                        //    .pathname
-                        //    .split('/')
-                        //    .last()
-                        //    .unwrap()
-                        //    .strip_prefix("docker-")
-                        //    .unwrap()
-                        //    .strip_suffix(".scope")
-                        //    .unwrap();
```

Comment:
Is it OK if we remove this? As it got moved to `extract_pod_id_from_cgroup_path`.